docs/blog/newsletter-november-2021.rst

```diff
@@ -0,0 +1,137 @@
+
+---------------------------
+Options for single-sourcing
+---------------------------
+
+Single-source publishing is a content management technique that lets you use one source of content but publish it to different output formats (“write once, publish everywhere”). It's not something that every team needs, but it can be super useful if you need to publish to the web *and* PDFs, or if you have many similar variants on one product. If your team is not “technical” - more specifically if they are unfamiliar with version control software like Git, lack software development skills, or have no way of obtaining the support necessary to implement a docs-as-code approach, component content management systems (CCMS) like Paligo and MadCap Flare might be more suitable.
+
+Regardless of the method you choose, decide on a single-source publishing strategy and consider the advantages and disadvantages of each option to determine which is ideal for your team. Distributed contributions, for example, are maybe a reason to choose the docs-as-code method. In contrast, if you use Flare or a docs-focused CMS like Knowledge Owl, you'll find more robust support for reusing articles or snippets. Whatever method or tool you choose, have a plan in place to ensure a successful single-source publishing approach.
+
+--------------------------------------------------
+Documentation gets respect (or at least attention)
+--------------------------------------------------
+
+Two recent reports highlight the increased attention documentation seems to be getting in the worlds of software development.
+
+The `2021 State of DevOps report from Google Cloud <https://cloud.google.com/devops/state-of-devops>`__ includes data showing the effects of docs quality on devops team performance, and describes practices to improve the quality of your documentation. The report is based on seven years of research and data from more than 32,000 professional worldwide, focusing on "capabilities and practices that drive software delivery, operation, and organizational performance." So the focus is not on docs -- but the report includes good documentation as one of six findings characteristic of the 26% of teams studied who fall into the highest performing category. This is _internal_ documentation, and its quality has a statistically significant impact on how secure, reliable, and effective a team's devops practices are. Quality documentation efforts should include critical use cases for products and services, guidelines for updating and editing existing documentation, clearly defined ownership and responsibility for documentation, inclusion of docs as part of the software development process, and recognition of docs work during performance reviews and promotions.
```

Comment:
```suggestion
The `2021 State of DevOps report from Google Cloud <https://cloud.google.com/devops/state-of-devops>`__ includes data showing the effects of docs quality on devops team performance, and describes practices to improve the quality of your documentation. The report is based on seven years of research and data from more than 32,000 professionals worldwide, focusing on "capabilities and practices that drive software delivery, operation, and organizational performance." So the focus is not on docs -- but the report includes good documentation as one of six findings characteristic of the 26% of teams studied who fall into the highest performing category. This is _internal_ documentation, and its quality has a statistically significant impact on how secure, reliable, and effective a team's devops practices are. Quality documentation efforts should include critical use cases for products and services, guidelines for updating and editing existing documentation, clearly defined ownership and responsibility for documentation, inclusion of docs as part of the software development process, and recognition of docs work during performance reviews and promotions.
```

Reply:
OK wow the suggestion here is not highlighted for some reason -- it's adding an `s` to "professional" in "...more than 32,000 professional worldwide..."